test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -788,7 +790,7 @@ public void GenerateSchema_HonorsSerializerAttribute_JsonRequired()
             var referenceSchema = Subject().GenerateSchema(typeof(JsonRequiredAnnotatedType), schemaRepository);
 
             var schema = schemaRepository.Schemas[referenceSchema.Reference.Id];
-            Assert.Equal(new[] { "StringWithConflictingRequired", "StringWithJsonRequired"}, schema.Required.ToArray());
+            Assert.Equal(new[] { "StringWithConflictingRequired", "StringWithJsonRequired" }, schema.Required.ToArray());
```

Comment:
Can this use collection expressions too?

```suggestion
            Assert.Equal(["StringWithConflictingRequired", "StringWithJsonRequired"], schema.Required);
```